gui/src/containers/JobSpecRuns.js

```diff
@@ -38,28 +84,31 @@ export class JobSpecRuns extends Component {
   }
 
   componentDidMount () {
-    const {jobSpecId, pageSize, fetchJobSpecRuns} = this.props
-    fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+    const { jobSpecId, pageSize, fetchJobSpecRuns } = this.props
+    if (this.props.match.params.jobRunsPage) {
+      const START_PAGE = this.props.match.params.jobRunsPage
+      this.setState({ page: START_PAGE - 1 })
+      fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+    } else {
+      this.setState({ page: 0 })
+      fetchJobSpecRuns(jobSpecId, 1, pageSize)
```

Comment:
Same conversation as https://github.com/smartcontractkit/chainlink/pull/457 about the juggling of 0 based and 1 based page numbers.